src/traces/choroplethmapbox/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    attributes: require('./attributes'),
+    supplyDefaults: require('./defaults'),
+    colorbar: require('../heatmap/colorbar'),
+    calc: require('../choropleth/calc'),
+    plot: require('./plot'),
+    hoverPoints: require('../choropleth/hover'),
+    eventData: require('../choropleth/event_data'),
+    selectPoints: require('../choropleth/select'),
+
+    style: function(_, cd) {
+        if(cd) {
+            var trace = cd[0].trace;
+            trace._glTrace.updateOnSelect(cd);
+        }
+    },
+
+    moduleType: 'trace',
+    name: 'choroplethmapbox',
+    basePlotModule: require('../../plots/mapbox'),
+    categories: ['mapbox', 'gl', 'noOpacity'],
```

Comment:
Later on the road, it may be of interest to have traces with opacity. Right?

Reply:
`opacity` here for `choroplethmapbox` trace would be equivalent to `marker.opacity`, so we don't need it.

Reply:
What about dimming marker.lines? Trace opacity may be handy for adjusting both.

Reply:
Maybe, but I'm not going to do this now for consistency with `choropleth`:

https://github.com/plotly/plotly.js/blob/cb202a8e47631e20555de382d2bbc7393625519b/src/traces/choropleth/index.js#L26